src/test/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImplTest.java

```diff
@@ -94,49 +82,31 @@ void getMediaMetadataShouldReturnRepositoryResultsUnmodifiedWhenRepositoryResult
 
     @Test
     void saveProcessedDataShouldSaveBlobAndSetStatus() {
-        final String containerName = "ContainerName";
-
         final MediaRequestEntity mediaRequestEntity = new MediaRequestEntity();
         final MediaRequestEntity mediaRequestEntityUpdated = new MediaRequestEntity();
         mediaRequestEntityUpdated.setStatus(COMPLETED);
 
-        when(mockDataManagementConfiguration.getOutboundContainerName()).thenReturn(containerName);
-
-        when(mockDataManagementService.saveBlobData(
-            any(),
-            any()
-        )).thenReturn(BLOB_LOCATION);
+        when(mockDataManagementApi.saveBlobDataToOutboundContainer(any()))
+            .thenReturn(BLOB_LOCATION);
 
         when(mockTransientObjectDirectoryService.saveTransientDataLocation(
             any(),
             any()
         )).thenReturn(mockTransientObjectDirectoryEntity);
 
-        when(mockMediaRequestService.updateAudioRequestStatus(
-             any(),
-             any()
-         )).thenReturn(mediaRequestEntityUpdated);
-
         audioTransformationService.saveProcessedData(
             mediaRequestEntity,
             BINARY_DATA
         );
 
-        verify(mockDataManagementService).saveBlobData(
-            eq(containerName),
+        verify(mockDataManagementApi).saveBlobDataToOutboundContainer(
             eq(BINARY_DATA)
```

Comment:
Would you be able to remove the `eq` I added here - looks like I did that previously
but isn't needed according to Sonar - clumsy code smell - Apologies
https://sonarcloud.io/project/issues?resolved=false&id=uk.gov.hmcts.reform%3Adarts-api&open=AYlTeaKEuvsei9NxDHhV

Reply:
Done